### PR TITLE
Arrow 0.11.1

### PR DESCRIPTION
**Pull request by kszucs**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kszucs** [comment]
@conda-forge-admin please rerender

**Review by xhochy — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -3,14 +3,14 @@
 
 # https://github.com/apache/arrow/blob/master/cpp/src/parquet/.parquetcppversion
 {% set parquet_version = "1.5.1" %}
-{% set arrow_version = "0.11.0" %}
+{% set arrow_version = "0.11.1" %}
```
Comment (wesm) [inline recipe/meta.yaml]: I think this should not be pinned. Can you change this to a `>=` condition? Ideally we will not be altering this package any more

Comment (kszucs) [reply recipe/meta.yaml]: Indeed! Done. If We maintain a separate parquet-cpp version then this recipe still needs to be updated occasionally.

**Comment by kszucs** [comment]
So I've only changed the condition to >=0.11.0 without bumping arrow-cpp's version.

**Comment by wesm** [comment]
+1. go ahead and merge once the build passes

**Comment by kszucs** [comment]
Ok, will do.

**Pull request merged**